packages/terra-core-docs/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 * Updated
   * Updated `terra-scroll` to add realistic examples.
+  *  Updated all paginator examples to use meaningful titles.
+  *  Updated testing recommendations to use `jest.spyOn` instead of `jest.mock`.
```

Comment:
Updated with main & piggy-backing off this PR to moved this line to the correct section.

Reply:
Thank you for resolving the conflicts!